hack/dev-env/start-principal.sh

```
@@ -46,7 +46,23 @@ if [ -f "$E2E_ENV_FILE" ]; then
     export ARGOCD_PRINCIPAL_ENABLE_WEBSOCKET=${ARGOCD_PRINCIPAL_ENABLE_WEBSOCKET:-false}
 fi
 
+# Set a longer informer sync timeout for E2E tests (default is 60s, use 120s for CI)
+# export ARGOCD_PRINCIPAL_INFORMER_SYNC_TIMEOUT=${ARGOCD_PRINCIPAL_INFORMER_SYNC_TIMEOUT:-120s}
+
 SCRIPTPATH="$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
```

Shellcheck (0.11.0)

[warning] SCRIPTPATH appears unused. Verify use (or export if used externally).

(SC2034)